src/core/mantle/types/index.ts

```diff
@@ -70,12 +71,12 @@ export type DataRange = {
 export type DataType = "geojson" | "3dtiles";
 
 // Feature
-
 export type Feature = {
   id: string;
   geometry?: Geometry;
   properties?: any;
   range?: DataRange;
+  [key: string]: any;
```

Comment:
```suggestion
```

Variable can refer to `id` and fields in `properties` of a Feature, not Feature itserlf.